test/configuration/displayname.spec.conf/displayname_schema.json

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
general comment here. this seems like a huge schema just to test display name. i think these should be much smaller scoped and focused so that it just covers the schema needed for the test case at hand. i would expect a couple tables at most so that you can test the display name annotation in the primary entity and in nested entities.